corehq/apps/change_feed/tests/test_pillow.py

```diff
@@ -122,6 +128,16 @@ def test_mismatched_rev(self):
                 )
             )
 
+        with self.assertRaises(DocumentMismatchError):
```

Comment:
I might just be overlooking something but shouldn't there be a test that explicitly checks that something smaller does not raise an exception?